src/NuGet.Core/NuGet.Packaging/Core/PackagingException.cs

```diff
@@ -6,6 +6,7 @@
 
 namespace NuGet.Packaging.Core
 {
+    [SerializableAttribute]
     public class PackagingException : Exception, ILogMessageException
```

Comment:
This class needs to implement `ISerializable` properly for `_logMessage` to round trip